docs/docs/en/guides/develop/build/registry.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Registry
+titleTemplate: :title · Build · Develop · Guides · Tuist
+description: Optimize your Swift package resolution times by leveraging the Tuist Registry.
+---
+
+# Registry {#registry}
+
+> [!WARNING] BETA
+> This feature is currently in beta. If you encounter any issues, please report them at our <LocalizedLink href="https://community.tuist.dev/c/troubleshooting-how-to/6">community forum</LocalizedLink>.
```

Comment:
You don't need a localized link here. Those are for links to other documentation pages and the component ensures that the locale information is preserved:

```suggestion
> This feature is currently in beta. If you encounter any issues, please report them at our <a href="https://community.tuist.dev/c/troubleshooting-how-to/6" target="_blank">community forum</a>.
```